.github/cloud_code_issue.md

```diff
@@ -1,3 +1,12 @@
+---
+name: Cloud Code Report
```

Comment:
Can we more specific? name: Cloud Code <plugin?>Report

Reply:
updated